README.md

```diff
@@ -6,21 +6,22 @@
 
 ## ✨ Features
 
-* open links without `netrw`
```

Comment:
why did you change these all to hyphens ?

Reply:
sorry about that, seems like my LSP formatted the markdown file

I'll revert the changes and fix